pr_testing/setup-pr-test-env.sh

```diff
@@ -59,9 +59,10 @@ if [ "${CMSSW_CVMFS_PATH}" != "" ] ; then
   popd
   mkdir -p $WORKSPACE/${CMSSW_VERSION}
   if [ -f ${CMSSW_CVMFS_PATH}/ibeos_cache.txt ] ; then ln -s ${CMSSW_CVMFS_PATH}/ibeos_cache.txt $WORKSPACE/${CMSSW_VERSION}/ibeos_cache.txt ; fi
-  ln -s ${CMSSW_CVMFS_PATH}/src $WORKSPACE/${CMSSW_VERSION}/src
-  cp -r ${CMSSW_CVMFS_PATH}/.SCRAM $WORKSPACE/${CMSSW_VERSION}/.SCRAM
-  cp -r ${CMSSW_CVMFS_PATH}/config $WORKSPACE/${CMSSW_VERSION}/config
+  cp -r ${CMSSW_CVMFS_PATH}/{src,config,.SCRAM} $WORKSPACE/${CMSSW_VERSION}/
+  pushd $WORKSPACE/${CMSSW_VERSION}/
+    scram b ProjectRename
+  popd
```

Comment:
@iarspider , this might work. Did you check if you really need to copy `src and config` ? Can we live with just symlinks?

Reply:
humm, this might break all other tests as in this case local directory will be missing all the libs and bins tests

Reply:
I was hoping that we can just symlink `src` and copy everything else, but I couldn't tell scram where to put `tmp` dir - it was trying to put it on `/cvmfs`, which obviously fails.

Reply:
We can pass a flag to this script to only do copying for gpu unit tests

Reply:
I think we can just run the `install.sh` to create a local copy of the build area and run unit test in that directory. This should be done only for unit tests, all other tests should use the area from cvmfs.

Reply:
for the pr unit test job, we can do
```
cd $WORKSPACE
$CMSSW_CVMFS_PATH/../install.sh
```
and this will create a local cmssw area. here we should be able to run unit tests

Reply:
@smuzaffar do I need to run `scram b ProjectRename` after `install.sh`?

Reply:
No, `install.sh` internally takes care of it.